app/components/tab_navigation_component.rb

```diff
@@ -28,14 +28,16 @@ def current_path?(path)
 
   def nav_list_item(route, &block)
     if current_path?(route[:path])
-      render(
-        ClickObserverComponent.new(
-          event_name: 'tab_navigation_current_page_clicked',
-          payload: { path: route[:path] },
-          role: 'listitem',
```

Comment:
Am I correct in my understanding that `role: 'listitem'` is being removed because the content tag is a `<li>`?